src/main/java/de/teamlapen/vampirism/entity/ai/EntityAIUseAction.java

```diff
@@ -0,0 +1,149 @@
+package de.teamlapen.vampirism.entity.ai;
+
+import de.teamlapen.vampirism.api.difficulty.IAdjustableLevel;
+import de.teamlapen.vampirism.api.entity.actions.DefaultEntityAction;
+import de.teamlapen.vampirism.api.entity.actions.IEntityAction;
+import de.teamlapen.vampirism.api.entity.actions.IInstantAction;
+import de.teamlapen.vampirism.api.entity.actions.ILastingAction;
+import de.teamlapen.vampirism.api.entity.factions.IFactionEntity;
+import de.teamlapen.vampirism.entity.EntityVampirism;
+import net.minecraft.entity.ai.EntityAIBase;
+import net.minecraft.entity.player.EntityPlayer;
+import java.util.List;
+import java.util.Random;
+
+/**
+ * Usage for every {@link IFactionEntity} like Hunter/Vampire entities
+ * can be used as AITask in custom task list or with {@link handle()} in UpdateLiving in an Entity
+ */
+public class EntityAIUseAction<T extends EntityVampirism & IFactionEntity & IAdjustableLevel> extends EntityAIBase {
```

Comment:
Currently only EntityBasicVampire makes use of this, but not in the AITask, but with the handle() method.
Are you actually planning to use it in the AITask for certain entities?

If not, you could consider migrating away from the AI concept and do something similar like the ActionHandler and IActionPlayer for the entity action.